test/utils/Signer.py

```diff
@@ -12,14 +12,16 @@ def sign(self, message_hash):
 
     async def send_transaction(self, account, to, selector_name, calldata, nonce=None):
         if nonce is None:
-            nonce, = await account.get_nonce().call()
+            execution_info = await account.get_nonce().call()
+            nonce, = execution_info.result
 
         selector = get_selector_from_name(selector_name)
         message_hash = hash_message(
             account.contract_address, to, selector, calldata, nonce)
         sig_r, sig_s = self.sign(message_hash)
+        signatures = [sig_r, sig_s]
```

Comment:
The whole array is a single signature, so I'd call it just `signature`. Maybe even just pass the literal array to the funciton below.